library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -169,31 +169,31 @@ sealed class Conversation {
         }
     }
 
-    fun send(prepared: PreparedMessage): String {
+    suspend fun send(prepared: PreparedMessage): String {
```

Comment:
Is this a breaking change, or will existing code continue to work (albeit no longer waiting for the publish to complete)? Not in this PR, but long-term do we have any thoughts on splitting out the two conversation types, and allowing the API's to be different across the two to maintain backwards compatibility?

Reply:
Technically it's a breaking change if anyone was using send outside of a thread. As you can see there are no changes to the example app in this PR because my example app was already calling send in a thread so I honestly think this change may be unnoticeable to most integrators as they were likely sending it asynchronously anyways.